### PR TITLE
1.6.x -> master (analysis view)

### PR DESCRIPTION
**Pull request by MetRonnie**
Partially addresses https://github.com/cylc/cylc-ui/issues/1279

Includes:
- #1254

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Tests are updated
- [x] Changelog entry brought through
- [x] No docs needed

### CODE REVIEW

**Comment by MetRonnie** [comment]
The real changes here are in https://github.com/cylc/cylc-ui/pull/1308/commits/7cb7ab4582909a7befd6e9fa0f56155e082a11aa

File: tests/e2e/specs/analysis.cy.js [hunk 1]
```diff
@@ -16,16 +16,19 @@
  */
 
 describe('Analysis view', () => {
-  it('Should display the mocked workflow', () => {
+  beforeEach(() => {
     cy.visit('/#/analysis/one')
+  })
```
Comment (oliver-sanders) [inline tests/e2e/specs/analysis.cy.js]: FYI, the intention was to support other workflows in the offline data, which we may still do to help test views for which the simple example isn't enough. This view is a good example of one which could benefit from a more advanced example.

It's also not a bad idea to test a views interaction with the workspace page as we will add more interactions between views in due course. E.G. we already have the ability to open log files in a new tab, we should also acquire the ability to select a task in one view and use that to find the same task in another view (e.g. click+drag type functionality), keyboard controls, that sort of stuff.

No need to go collecting `cy.visit`s in all tests is the point, we may end up reversing these diffs.

**Review by oliver-sanders — APPROVED** [review]
LGTM, tested.

**Review by wxtim — APPROVED** [review]
I've thoroughly tested it manually and had a quick flick through the code.

**Pull request merged**